src/odemis/util/conversion.py

```diff
@@ -502,17 +502,17 @@ def represent_numpy_floating(self, data):
 class YamlExtraDumper(Emitter, Serializer, CustomSafeRepresenter, Resolver):
 
     def __init__(self, stream,
-            default_style=None, default_flow_style=False,
-            canonical=None, indent=None, width=None,
-            allow_unicode=None, line_break=None,
-            encoding=None, explicit_start=None, explicit_end=None,
-            version=None, tags=None):
+                 default_style=None, default_flow_style=False,
+                 canonical=None, indent=None, width=None,
+                 allow_unicode=None, line_break=None,
+                 encoding=None, explicit_start=None, explicit_end=None,
+                 version=None, tags=None, **kwargs):
         Emitter.__init__(self, stream, canonical=canonical,
-                indent=indent, width=width,
-                allow_unicode=allow_unicode, line_break=line_break)
+                         indent=indent, width=width,
+                         allow_unicode=allow_unicode, line_break=line_break)
         Serializer.__init__(self, encoding=encoding,
-                explicit_start=explicit_start, explicit_end=explicit_end,
-                version=version, tags=tags)
+                            explicit_start=explicit_start, explicit_end=explicit_end,
+                            version=version, tags=tags)
         CustomSafeRepresenter.__init__(self, default_style=default_style,
-                default_flow_style=default_flow_style)
+                                       default_flow_style=default_flow_style)
```

Comment:
I think we should still pass "sort_keys" if it's received. Either just put a `**kwargs` here and hope for the best, or explicitly look in kwargs like that:
```python
kwargs_representer = {}
if "sort_keys" in kwargs:
    kwargs_representer["sort_keys"] = kwargs["sort_keys"]
CustomSafeRepresenter(..., **kwargs_representer)
```